app/src/main/res/layout-sw600dp-land/onboarding_fragment.xml

```diff
@@ -95,8 +96,9 @@
           android:text="@string/skip"
           android:textAllCaps="true"
           android:textColor="@color/oppiaPrimaryText"
-          android:textSize="14sp"
+          android:textSize="20sp"
           android:visibility="@{viewModel.slideNumber != (viewModel.totalNumberOfSlides - 1) ? View.VISIBLE: View.GONE, default=visible}"
+          app:layout_constraintBottom_toBottomOf="@+id/slide_dots_container"
           app:layout_constraintStart_toStartOf="parent"
           app:layout_constraintTop_toTopOf="@+id/slide_dots_container" />
```

Comment:
Color code for `onboarding_fragment_divider_view ` is `#707070` in the mocks and you are using `#333333` which is incorrect.

Reply:
Done